jdi-light/src/main/java/com/epam/jdi/light/actions/ActionHelper.java

```diff
@@ -21,57 +29,89 @@
+import static com.epam.jdi.light.logger.AllureLogger.takeHtmlCodeOnFailure;
+import static com.epam.jdi.light.logger.LogLevels.ERROR;
+import static com.epam.jdi.light.logger.LogLevels.INFO;
+import static com.epam.jdi.light.logger.LogLevels.STEP;
+import static com.epam.jdi.light.logger.Strategy.FAIL;
+import static com.epam.jdi.light.settings.JDISettings.getJDISettings;
+import static com.epam.jdi.light.settings.WebSettings.getWebSettings;
+import static com.epam.jdi.tools.JsonUtils.beautifyJson;
+import static com.epam.jdi.tools.LinqUtils.filter;
+import static com.epam.jdi.tools.LinqUtils.where;
+import static com.epam.jdi.tools.PrintUtils.print;
+import static com.epam.jdi.tools.ReflectionUtils.getAllFields;
+import static com.epam.jdi.tools.ReflectionUtils.isClass;
+import static com.epam.jdi.tools.ReflectionUtils.isInterface;
+import static com.epam.jdi.tools.StringUtils.LINE_BREAK;
+import static com.epam.jdi.tools.StringUtils.arrayToString;
+import static com.epam.jdi.tools.StringUtils.msgFormat;
+import static com.epam.jdi.tools.StringUtils.splitLowerCase;
+import static com.epam.jdi.tools.Timer.nowTime;
+import static com.epam.jdi.tools.map.MapArray.IGNORE_NOT_UNIQUE;
 import static com.epam.jdi.tools.map.MapArray.map;
-import static com.epam.jdi.tools.map.MapArray.*;
-import static com.epam.jdi.tools.pairs.Pair.*;
-import static com.epam.jdi.tools.switcher.SwitchActions.*;
-import static java.lang.Character.*;
+import static com.epam.jdi.tools.pairs.Pair.$;
+import static com.epam.jdi.tools.switcher.SwitchActions.Case;
+import static com.epam.jdi.tools.switcher.SwitchActions.Default;
+import static com.epam.jdi.tools.switcher.SwitchActions.Switch;
+import static com.epam.jdi.tools.switcher.SwitchActions.Value;
+import static java.lang.Character.toUpperCase;
 import static java.lang.String.format;
-import static java.lang.System.*;
-import static java.lang.Thread.*;
+import static java.lang.System.currentTimeMillis;
+import static java.lang.Thread.currentThread;
 import static java.util.Collections.reverse;
-import static org.apache.commons.lang3.StringUtils.*;
+import static org.apache.commons.lang3.StringUtils.isBlank;
 
 /**
  * Created by Roman Iovlev on 14.02.2018
  * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
  */
 public class ActionHelper {
+    private static final JDISettings jdiSettings = getJDISettings();
+    private static final WebSettings webSettings = getWebSettings();
+
     static String getTemplate(LogLevels level) {
-        if (LOGS.logInfoDetails != null) {
-            switch (LOGS.logInfoDetails) {
-                case NONE: return "{action}";
-                case NAME: return "{action} ({name})";
-                case LOCATOR: return "{action} ({locator})";
-                case CONTEXT: return "{action} ({context})";
-                case ELEMENT: return "{action} ({element})";
+        if (jdiSettings.LOGS.logInfoDetails != null) {
+            switch (jdiSettings.LOGS.logInfoDetails) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)